python/cudf/cudf/core/column/column.py

```diff
@@ -1526,6 +1540,15 @@ def build_column(
             null_count=null_count,
             children=children,
         )
+    elif is_interval_dtype(dtype):
+        return cudf.core.column.IntervalColumn(
+            data=data,
+            dtype=dtype,
+            mask=mask,
+            size=size,
+            offset=offset,
+            null_count=null_count,
```

Comment:
Do we need to pass `children` here as well?

Reply:
Yes, probably, I'll double-check this and update, unless it breaks anything.

Reply:
Yes, I'll add that in :D